### PR TITLE
styling added

### PR DESCRIPTION
**Pull request by imubarak234**
- You will find all the details of the Bookstore design in Zeplin.

- Login to Zeplin (hint: you will find the credentials in the sneak peek of the project).

- Open the project Bookstore CMS.

- Use the information detailed in Zeplin to style the website.

- Your final design should match Zeplin's design.

Have a good review 🤝

### CODE REVIEW

File: src/components/HeadNav.js [hunk 1]
```diff
@@ -1,27 +1,33 @@
...
           <ul className="navbar-nav">
             <li className="nav-item">
-              <NavLink className="nav-link" to="/">
+              <NavLink className="nav-link text-dark fw-normal pe-4" to="/">
                 Books
               </NavLink>
             </li>
             <li className="nav-item">
-              <NavLink className="nav-link" to="/categories">
+              <NavLink className="nav-link text-dark fw-normal pe-4" to="/categories">
```
Comment (Ghiftee) [inline src/components/HeadNav.js]: - ``` Categories ``` should not have the ```text-dark``` color. Kindly refer to the [Zeplin design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) for the correct color.

Comment (Ghiftee) [inline src/components/HeadNav.js:23]: - ``` Categories ``` should not have the ```text-dark``` color. Kindly refer to the [Zeplin design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) for the correct color.

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello 👋🏽 @imubarak234 

Good job working on the project so far! 
✔️ No linter errors
However, there are some issues that you need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._
- [ ] As it is very important that your design is exactly like the [Zeplin design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378), please ensure that you use the circle and not a square as shown below

-  Zeplin design
Screenshot 2022-01-22 at 12 17 16

- Your design 
Screenshot 2022-01-22 at 12 16 41

- [ ] Kindly ensure the measurements of your design are the same as the [Zeplin design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378)
- Zeplin design

Screenshot 2022-01-22 at 12 24 24

- Your design

Screenshot 2022-01-22 at 12 24 54




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/AddBooks.js [hunk 2]
```diff
@@ -13,23 +13,21 @@ const AddBooks = (props) => {
...
-          </label>
+    <div>
+      <form action="#" className="py-5 container" onSubmit={handleSubmit}>
+        <h3 id="form-title">ADD NEW BOOK</h3>
+        <div className="row gx-5 d-flex">
+          <div className="mb-2 mt-3 col-md-7">
+            <input type="text" id="titles" className="form-control p-3" placeholder="Book Title" name="title" required />
+          </div>
+          <div className="mt-3 col-md-3">
+            <input type="text" id="category" className="form-control p-3" placeholder="Category ▼" name="category" required />
```
Comment (rukundoeric) [inline src/components/AddBooks.js]: - [ ] <b>REQUIRED:</b> Category must be a selector not just text input. Kindly address that.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @imubarak234, 

_Great job so far 👏 💯 👍. You've done well implementing some of the requested changes. However, there are still some which aren't addressed yet. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 
  - Addressed some of the issues raised from the previous review. 💯 

## Required Changes ♻️
  - [ ] There are still a few things we need to fix before we move on.
  scrnli_1_22_2022_5-33-54 PM
       - There should be a small space between progress bar and the other content
       - The category must be a selector, not just text input.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
  
  - N/A 

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi @imubarak234 🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**